Makefile

```
@@ -765,12 +766,22 @@ pr\#%: clean-all
 	$(GO) run contrib/pr/checkout.go $*
 
 .PHONY: golangci-lint
-golangci-lint:
+golangci-lint: golangci-lint-check
+	golangci-lint run --timeout 10m
+
+.PHONY: golangci-lint-check
+golangci-lint-check:
+	$(eval GOLANGCI_LINT_VERSION := $(shell printf "%03d%03d%03d" $(shell golangci-lint --version | grep -Eo '[0-9]+\.[0-9.]+' | tr '.' ' ');))
 	@hash golangci-lint > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
+		echo "Downloading golangci-lint"; \
```

might be useful to add the version:

```suggestion
		echo "Downloading golangci-lint v${MIN_GOLANGCI_LINT_VERSION}"; \
```

---

That's useful, thanks!